### ISSUE TITLE
Change loss function

### ISSUE THREAD

**Issue by terrie1028** [dino-chiio/blip-vqa-finetune#7]
Hi, @dino-chiio.
First of all, thanks for sharing your code.

I have a question.
In the fine-tuning code, is it possible to replace the loss function with a loss other than that of the exsisting model?
If it is possible, I would appreciate it if you could tell me how.

Thank you!
